includes/polling/os/rutos-2xx.inc.php

```diff
@@ -0,0 +1,32 @@
+<?php
+$hardware = trim(snmp_get($device, '1.3.6.1.4.1.48690.1.7.0', '-OQv', '', ''), '"');
```

Comment:
trim() functions are not needed.

Reply:
I changed it completely to OIDs from MIB, they are didn't worked for me, but I found where was a problem.